tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "target": "es5",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "lib": [
+      "es6"
+    ],
+    "declaration": true,
+    "isolatedModules": true,
```

Comment:
change to false (https://www.typescriptlang.org/tsconfig#isolatedModules)

Reply:
Done, removed since default is false